tests/acceptance/test_sections_and_flows.py

```diff
@@ -0,0 +1,503 @@
+        page.locator(f'[test-id="{MARKER_START_SECTION}"]').first.click()
+        page.keyboard.press("ArrowDown")
+        page.keyboard.press("Enter")
+        # Select end section
+        page.locator(f'[test-id="{MARKER_END_SECTION}"]').first.click()
+        page.keyboard.press("ArrowDown")
+        page.keyboard.press("Enter")
+        page.locator(f'[test-id="{MARKER_DIALOG_APPLY}"]').first.click()
+        table = page.locator('[test-id="marker-flow-table"]').first
+        table.wait_for(state="visible")
+        # Wait until the table contains the newly created flow name
+        deadline_flow = time.time() + ACCEPTANCE_TEST_WAIT_TIMEOUT
+        while time.time() < deadline_flow:
+            try:
+                if custom_flow_name in table.inner_text():
+                    break
+            except Exception:
+                pass
+            time.sleep(0.1)
+        assert custom_flow_name in table.inner_text()
+        row = table.locator("tbody tr").filter(has_text=custom_flow_name).first
+        row.click()
+        page.locator('[test-id="marker-button-properties"]').first.click()
+        name_input = page.locator(f'[test-id="{MARKER_FLOW_NAME}"]').first
+        new_flow_name = "My-Flow-Renamed"
+        name_input.fill(new_flow_name)
+        page.keyboard.press("Escape")
+        time.sleep(0.2)
+        assert custom_flow_name in table.inner_text()
+        assert new_flow_name not in table.inner_text()
+        page.locator('[test-id="marker-button-properties"]').first.click()
+        name_input = page.locator(f'[test-id="{MARKER_FLOW_NAME}"]').first
+        name_input.fill(new_flow_name)
+        page.locator('[test-id="marker-apply"]').first.click()
+        time.sleep(0.2)
+        assert new_flow_name in table.inner_text()
+
+    @pytest.mark.skip(reason="only works in headed right now")
+    @pytest.mark.timeout(400)
+    @pytest.mark.playwright
+    @pytest.mark.usefixtures("external_app")
+    def test_remove_flow_and_generate_flows_and_save_project(
+        self,
+        page: Page,
+        external_app: NiceGUITestServer,
+        resource_manager: ResourceManager,
+        test_data_tmp_dir: Path,
+    ) -> None:
+        # Prepare and create sections
+        self._navigate_and_prepare(page, external_app, resource_manager)
+        self._goto_sections_with_one_video(page, resource_manager)
+        names = ["First-Line", "Second-Line"]
+        for n in names:
+            self._create_section(page, resource_manager, n)
+
+        # Create one flow then remove it
+        page.get_by_text(
+            resource_manager.get(FlowAndSectionKeys.TAB_FLOW), exact=True
+        ).click()
+        try:
+            page.get_by_text(
+                resource_manager.get(FlowKeys.BUTTON_ADD), exact=True
+            ).click()
+        except Exception:
+            page.locator('[test-id="marker-button-add"]').first.click()
+        page.locator(f'[test-id="{MARKER_FLOW_NAME}"]').first.fill("Temp-Flow")
```

Comment:
Introduce a helper to get an element for a given marker. Usage:
get_element(MARKER_FLOW_NAME).fill("...")